articles/test-runner-plugins.md

```diff
@@ -0,0 +1,236 @@
+<!--
+parent: 'Developer Guide'
+toc: true
+skip : true
+authors:
+    - 'Bertrand Chevrier'
+tags:
+    - 'Frontend'
+    - 'Plugin'
+    - 'JavaScript'
+    - 'Test Runner'
+-->
```

Comment:
You maybe want to experiment with tag groups too, see wiki, but this is entirely up to you